### ISSUE TITLE
Carthage support

### ISSUE THREAD

**Issue by TofPlay** [MengTo/Spring#202]
Hi,
For iOS and macOS projets it's will be great to make your component [Carthage](https://github.com/Carthage/Carthage) compatible

Take a look at these articles: 
- [Ruthlessly Simple Dependency Management with Carthage](https://realm.io/news/swift-dependency-management-with-carthage/)
- [Carthage Tutorial: Getting Started](https://www.raywenderlich.com/109330/carthage-tutorial-getting-started)
- [Making Carthage-compatible Frameworks with Swift](http://blog.mat.tc/how-to-make-a-carthage-compatible-framework/)
- [Creating Cross-Platform Swift Frameworks for iOS, watchOS, and tvOS via Carthage and CocoaPods] (http://basememara.com/creating-cross-platform-swift-frameworks-ios-watchos-tvos-via-carthage-cocoapods/) (👍 )

On your `README.md` add [![Carthage compatible](https://img.shields.io/badge/Carthage-Compatible-brightgreen.svg?style=flat)](https://github.com/Carthage/Carthage)  on top and add section for the installation with Carthage.

**Comment by codeservis** [comment]
Yeah i added inside cartfile your framework and when i adding custom class area DesignableView and dont shows Attribute Panel ? why ?

Ty
